pioneer/packages/page-staking/src/Overview/Address/index.tsx

```diff
@@ -130,6 +130,14 @@ function Address ({ address, className = '', filterName, hasQueries, isElected,
           <FormatBalance value={stakeOwn} />
         )}
       </td>
+      <td className='number'>
+        {(stakeOwn && stakeOther)
```

Comment:
`stakeOther` seems to be calculated based on `stakeTotal`, so perhaps we can just use the latter here and avoid too many unnecessary checks (only leaving one `stakeTotal ? /* ... */ : null`)